types/react/test/index.ts

```diff
@@ -467,6 +467,16 @@ type ForwardingRefComponentAsRef = React.ElementRef<typeof ForwardingRefComponen
 type MemoizedForwardingRefComponentAsRef = React.ElementRef<typeof MemoizedForwardingRefComponent>; // $ExpectType RefComponent
 type LazyComponentAsRef = React.ElementRef<typeof LazyComponent>; // $ExpectType RefComponent
 
+class CustomView extends React.Component { }
+declare global {
+    namespace JSX {
+        interface IntrinsicElements {
+            CustomView: IntrinsicClassAttributes<CustomView>;
```

Comment:
This won't work in runtime though right? All intrinsic elements has to start lowercase otherwise a JSX interpreter will interpret it as a symbol.

Reply:
Oh, react-pixi is using a capital case for its host elements. Should it be changed? Any documentation about it I can refer? Doesn't it handle React Native's host elements?

Reply:
Oh.. I see what you meant. It actually relies on the host implementation.